tests/conftest.py

```diff
@@ -26,21 +34,42 @@ def download_testdata():
     return
 
 
+def download_goes_testdata():
+    with requests.get(GOES_TESTDATA_URL, stream=True) as r:
+        with open(testdata_goes_dir / GOES_TESTDATA_URL.split("/")[-1], 'wb') as f:
+            shutil.copyfileobj(r.raw, f)
+    pass
```

Comment:
You don't need this :)

Reply:
the `pass` that is

Reply:
Ah crap. I often leave pass/return in when I write the function name days before the actual function. Removed now